### PR TITLE
Trigger doc build on specific changes

### PR DESCRIPTION
**Pull request by mishig25**

### CODE REVIEW

File: .github/workflows/delete_doc_comment.yml [hunk 1]
```diff
@@ -3,6 +3,9 @@ name: Delete dev documentation
 on:
   pull_request:
     types: [ closed ]
+    paths:
+      - "docs/**"
+      - "bindings/python/py_src/**"
```
Comment (mishig25) [inline .github/workflows/delete_doc_comment.yml:8]: combination of `types` & `paths` should work right?

Comment (Narsil) [reply .github/workflows/delete_doc_comment.yml:8]: It should yes. It's an OR here.

Comment (mishig25) [reply .github/workflows/delete_doc_comment.yml:8]: oh, I was expecting AND

in this case https://github.com/huggingface/safetensors/pull/88

Comment (Narsil) [reply .github/workflows/delete_doc_comment.yml:8]: Ohhh sorry, types and paths is and AND, however multiples paths is OR between each other.

**Pull request merged**